_posts/2019-09-09-what-is-mtls.md

```diff
@@ -4,25 +4,86 @@ current: post
+**Note** there are some [tutorials](#how-to-implement-mtls-some-tutorials-by-wott) within this article. These vary in difficulty. All require a linux distribution to be compatible with our agent.
 
-## Why is mTLS authentication important?
-IoT security rests on the proper management of machine identities. But, machine identities can be efficiently managed with the use of digital certificates and mTLS. mTLS authentication validates the trust between your device and your server by simultaneously identifying each other as authentic. That is a proper way for connection and data transmission to be established. If the initial validation process fails, access is denied and a connection is not made.
+Cybersecurity is an ever-growing, ever-complicating field. As a new developer, it can be daunting to think of where to start. Here we're going to talk a little about mTLS and why you as a developer should care about it (and implement it!). 
 
-## What can go wrong if you do not use mTLS?
-Not using mTLS may compromise your brand, your company, and your services in a multitude of ways. Consider a scenario where hackers compromise your device and use a commonly shared API key stored on disk to gain access to your back-end servers. This means that any sensitive information stored on your server might become accessible. Without mutual authentication there is no way to confirm client-side identity and isolate the compromised device. Imagine again, the occasion where without mTLS hackers take over a device and start sending falsified data. The data received and transmitted is altered, compromised and unreliable. But the users have no way of knowing this and they keep making critical decisions based on the compromised data received. They analyze metrics, they receive real-time data but…the integrity of the data is compromised so nothing can be trusted. What if the data compromised is related to health and smart healthcare devices? Or if it’s logistics-related data or even manufacturing and smart transportation data? What kind of a cataclysmic or disastrous impact will compromised data have? This is why in IoT infrastructure mTLS is an important piece of the security puzzle. IoT security is all about authenticating, trusting and securing the services provided.
+mTLS is just a type of TLS (Transport Layer Security). This is commonly found in verifying endpoints and for encrypting communications and also browsers. The 'https' in a url is a sign of a TLS controlled website. This part of the url verifies the identity of the server, the website you're accessing, to you, the client. 
```

Comment:
* "mTLS is just a type of TLS" - I'd change that to say that it is an *extension* of TLS.
* url -> URL
* "TLS controlled" - it's using TLS. It sounds strange to say that it is controlled.
* Clarification: mTLS is *very* rarely used in browsers. There are extension to enable this, but to my knowledge no browsers support it out-of-the-box.